float64 speed
float64 acc
float64 speedFiltered
float64 accFiltered